src/components/color_picker/utils.ts

```diff
@@ -156,3 +156,73 @@ export const getChromaColor = (input?: string | null, allowOpacity = false) => {
   }
   return null;
 };
+
+// Given an array of objects with key value pairs stop/color
+// Or given just an array of hex colors returns a css linear-gradient
+export const getLinearGradient = (palette: any) => {
```

Comment:
```suggestion
export const getLinearGradient = (palette: string[] | ColorStop[]) => {
```

I think this is correct given the comment